vita3k/gui/src/settings_dialog.cpp

```diff
@@ -750,13 +750,13 @@ void draw_settings_dialog(GuiState &gui, HostState &host) {
         if (ImGui::IsItemHovered())
             ImGui::SetTooltip("Save color surfaces to files.");
         ImGui::Spacing();
-        ImGui::Checkbox("Dump textures", &host.cfg.dump_textures);
+        ImGui::Checkbox("Texture dumping", &host.cfg.dump_textures);
         if (ImGui::IsItemHovered())
-            ImGui::SetTooltip("Dump textures to files");
+            ImGui::SetTooltip("Dumps textures to files");
         ImGui::SameLine();
-        ImGui::Checkbox("Dump elfs", &host.kernel.debugger.dump_elfs);
+        ImGui::Checkbox("ELF dumping", &host.kernel.debugger.dump_elfs);
```

Comment:
Same here as above, either "Dump ELFs" or change the rest of the debug tab to use that kind of sentece structure

Reply:
change to "ELF Dumping" to use same sentence structure as the rest checkboxes and should be good to merge

Reply:
I did this